src/Xamarin.Legacy.Sdk/Sdk/AutoImport.Android.props

```diff
@@ -10,7 +10,8 @@ NOTE! everything must be conditioned behind:
     <Using Include="Android.OS.Bundle" Alias="Bundle" Platform="Android" Sdk="Xamarin.Legacy.Sdk" />
   </ItemGroup>
 
-  <ItemGroup Condition=" ('$(TargetPlatformIdentifier)' == 'android' or $(TargetFramework.StartsWith ('MonoAndroid', StringComparison.OrdinalIgnoreCase))) and '$(EnableDefaultXamarinLegacySdkItems)' == 'true' ">
+  <ItemGroup Condition=" ('$(TargetPlatformIdentifier)' == 'android' or $(TargetFramework.StartsWith ('MonoAndroid', StringComparison.OrdinalIgnoreCase))) and '$(EnableDefaultXamarinLegacySdkItems)' == 'true' 
+                        and ('$(MonoAndroidAssetsPrefix)' != '' and '$(MonoAndroidResourcePrefix)' != '') ">
```

Comment:
Can we make an `<ItemGroup>` block for `$(MonoAndroidAssetsPrefix)` and one for `$(MonoAndroidResourcePrefix)`?

There is probably a case where one could be blank and not the other.